doc/faq.rst

```diff
@@ -207,3 +207,12 @@ temporarily disable your antivirus software to find out if it is the cause for
 your performance problems. If you are certain that the antivirus software is
 the cause for this and you want to gain maximum performance, you have to add
 the restic binary to an exclusions list within the antivirus software.
+
+How do I choose a strong password?
+----------------------------------
+
+Any password close to 20-30 characters should be  enough.  However,length
+is not the only component for strong passwords.A strong password includes
+Alphabetical, Numerical and Special characters. For example, 
```

Comment:
The single most important part is actually the password length. One can actually trade e.g. special characters and even upper case letters for an approx. 30% higher password length. I'd like to de-emphasize the different character types part a bit.

Reply:
Revising now.